### PR TITLE
Adding Paypal. 

### PR DESCRIPTION
**Pull request by kaddynator**
PayPal is a widely-used online payment system that allows individuals and businesses to send and receive money electronically. It offers a secure and convenient alternative to traditional payment methods like checks and money orders. We use airflow for date pipelines and in ML platform team.

### CODE REVIEW

**Comment by kaddynator** [comment]
@RNHTTR !

File: INTHEWILD.md [hunk 1]
```diff
@@ -372,7 +372,7 @@ Currently, **officially** using Airflow:
 1. [Paxful](https://paxful.com) [[@ne1r0n](https://github.com/ne1r0n)]
 1. [PayFit](https://payfit.com) [[@pcorbel](https://github.com/pcorbel)]
 1. [PAYMILL](https://www.paymill.com/) [[@paymill](https://github.com/paymill) & [@matthiashuschle](https://github.com/matthiashuschle)]
-1. [PayPal](https://www.paypal.com/) [[@r39132](https://github.com/r39132) & [@jhsenjaliya](https://github.com/jhsenjaliya)]
+1. [PayPal](https://www.paypal.com/)  [@kaddynator](https://github.com/kaddynator)], [[@r39132](https://github.com/r39132) & [@jhsenjaliya](https://github.com/jhsenjaliya)]
```
Comment (RNHTTR) [inline INTHEWILD.md]:
```suggestion
1. [PayPal](https://www.paypal.com/)  [[@kaddynator](https://github.com/kaddynator)], [@r39132](https://github.com/r39132) & [@jhsenjaliya](https://github.com/jhsenjaliya)]
```

**Review by RNHTTR — CHANGES_REQUESTED** [review]

File: INTHEWILD.md [hunk 2]
```diff
@@ -377,7 +377,7 @@ Currently, **officially** using Airflow:
 1. [Paxful](https://paxful.com) [[@ne1r0n](https://github.com/ne1r0n)]
 1. [PayFit](https://payfit.com) [[@pcorbel](https://github.com/pcorbel)]
 1. [PAYMILL](https://www.paymill.com/) [[@paymill](https://github.com/paymill) & [@matthiashuschle](https://github.com/matthiashuschle)]
-1. [PayPal](https://www.paypal.com/) [[@r39132](https://github.com/r39132) & [@jhsenjaliya](https://github.com/jhsenjaliya)]
+1. [PayPal](https://www.paypal.com/)  [[@kaddynator](https://github.com/kaddynator)], [@r39132](https://github.com/r39132) & [@jhsenjaliya](https://github.com/jhsenjaliya)]
```
Comment (RNHTTR) [inline INTHEWILD.md]:
```suggestion
1. [PayPal](https://www.paypal.com/)  [[@kaddynator](https://github.com/kaddynator), [@r39132](https://github.com/r39132) & [@jhsenjaliya](https://github.com/jhsenjaliya)]
```

**Review by RNHTTR — CHANGES_REQUESTED** [review]

File: INTHEWILD.md (hunk 2 above)
Comment (jedcunningham) [inline INTHEWILD.md]:
```suggestion
1. [PayPal](https://www.paypal.com/) [[@kaddynator](https://github.com/kaddynator)], [@r39132](https://github.com/r39132) & [@jhsenjaliya](https://github.com/jhsenjaliya)]
```

**Review by RNHTTR — APPROVED** [review]

File: INTHEWILD.md [hunk 3]
```diff
@@ -383,7 +383,7 @@ Currently, **officially** using Airflow:
 1. [Paxful](https://paxful.com) [[@ne1r0n](https://github.com/ne1r0n)]
 1. [PayFit](https://payfit.com) [[@pcorbel](https://github.com/pcorbel)]
 1. [PAYMILL](https://www.paymill.com/) [[@paymill](https://github.com/paymill) & [@matthiashuschle](https://github.com/matthiashuschle)]
-1. [PayPal](https://www.paypal.com/) [[@r39132](https://github.com/r39132) & [@jhsenjaliya](https://github.com/jhsenjaliya)]
+1. [PayPal](https://www.paypal.com/) [[@kaddynator](https://github.com/kaddynator)], [@r39132](https://github.com/r39132) & [@jhsenjaliya](https://github.com/jhsenjaliya)]
```
Comment (RNHTTR) [inline INTHEWILD.md]:
```suggestion
1. [PayPal](https://www.paypal.com/) [[@kaddynator](https://github.com/kaddynator), [@r39132](https://github.com/r39132) & [@jhsenjaliya](https://github.com/jhsenjaliya)]
```

**Review by RNHTTR — CHANGES_REQUESTED** [review]

**Review by RNHTTR — APPROVED** [review]

**Review by amoghrajesh — APPROVED** [review]
LGTM +1
@RNHTTR observed that some places have the format: `a,b & c` while some have `a, b, c`. Can be unified

**Comment by eladkal** [comment]
> @RNHTTR observed that some places have the format: a,b & c while some have a, b, c. Can be unified

We can always re-organize this if needed. I'll merge when CI is green

**Review by eladkal — APPROVED** [review]

**Pull request merged**